alfasim_sdk_api/detail/bootstrap_linux.h

```diff
@@ -10,74 +10,69 @@
 
 #define MAX_PATH 4096
 
-struct alfasim_sdk_bootstrap {
-    alfasim_sdk_bootstrap()
-    {
-        char SO_FILENAME[] = "/alfasim_plugins_api.so";
+inline void alfasim_sdk_open(ALFAsimSDK_API* api)
```

Comment:
Shouldn't we have the equivalent file for Windows?

Reply:
Forgot to add. It'll fail on Windows when I try to run the job hehehe